sql/create_membership_period.sql

```diff
@@ -0,0 +1,22 @@
+SET FOREIGN_KEY_CHECKS=0;
```

Comment:
maybe better to create a new directory under sql and rename the file so :  sql/Upgrader/0002_create_membership_period.sql

to make a better seperation